app/models/user.rb

```diff
@@ -977,6 +981,23 @@ def paypal_payout_email
     paypal_connect_account.paypal_account_details&.dig("primary_email")
   end
 
+  def purchased_small_bets?
```

Comment:
@xrav3nz @ershad

Reply:
FYI still making some changes here and in React.